basics/email_validation.php

```diff
@@ -0,0 +1,24 @@
+<?php
+require 'Creds.php';
+require 'requests.php';
+/**
+ * Creating object of Creds class to get the api key stored from the Creds class
+ */
+
+$creds = new Creds();
+$access_key = $creds->getApiKey();
+/**
```

Comment:
/**
         * Checks if the email is valid or not
         *
         * @param string $email_address
         *   stores email address
         *
         * @return bool
         */